### DISCUSSION TITLE
Adding a script yields surprising `chezmoi status` output

### DISCUSSION THREAD

**Discussion by moritzreiter** [twpayne/chezmoi#1762] [Q&A] [answered]
I added a run script to my chezmoi source directory [as described in the docs](https://www.chezmoi.io/docs/how-to/#use-scripts-to-perform-actions).

It works fine, but since then I get this:

```
❯ chezmoi status
 R mac-os-config
```

What does this mean? That the file was renamed? But I didn't rename it, I added and already committed it. Will this entry be forever in my `chezmoi status` output?

I just started using chezmoi, so maybe I'm misunderstanding something, but to my current understanding this seems to be wrong.

**Comment by twpayne** [comment, marked as answer]
`chezmoi status` tells you what will happen if you run `chezmoi apply`. In this case, it's telling you that `chezmoi apply` will run the `mac-os-config` script.

The conditions `mac-os-config` is run depend on whether it is a `run_` script, a `run_once_` script, or `run_onchange_` script.

**Reply by moritzreiter** [reply]
Thanks a lot for the explanation!

So the `R` actually stands for "run" and not for "rename" as it does for `git status`?

Is there a reference somewhere for the different codes and their meaning? I assumed they would be the same as for `git status`, because all I could found in the docs was a [short paragraph](https://www.chezmoi.io/docs/reference/#status) that refers to `git status`.

**Comment by twpayne** [comment]
Yes, the `R` stands for run. There's no reference for the codes, but you can [look at the code](https://github.com/twpayne/chezmoi/blob/v2.9.4/internal/cmd/statuscmd.go#L83-L102). `chezmoi status` only aims to be similar to `git status`. They cannot be exactly the same as the two programs do different things.

**Reply by moritzreiter** [reply]
Got it. Again, thanks a lot!
